portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```
@@ -66,8 +70,25 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String text = request.getParameter("form-comment"); 
-    String name = request.getParameter("form-name"); 
+    String body = "";
+    try (BufferedReader reader = request.getReader()) {
+      if (reader == null) {
+            return;
+      }
+      String line;
+      while ((line = reader.readLine()) != null) {
+        body = line; 
+      }
+    } catch (Exception e) {
+      System.out.println("oh no");
```

Instead of silently failing here, consider report an error back to the user (400).